snuba/datasets/cdc/groupedmessage_processor.py

```diff
@@ -0,0 +1,39 @@
+
+from dateutil.parser import parse as dateutil_parse
+from snuba.datasets.cdc.cdcprocessors import CdcProcessor
+
+
+class GroupedMessageProcessor(CdcProcessor):
```

Comment:
Doesn't this need to have a delete handler?

Reply:
At first glance it seemed to me deletion was always done by changing the status of the group record and then by periodically removing all deleted groups (action that would work differently in clickhouse). But I agree, adding explicitly a delete handler will keep the model consistent. Will fix